images/debian.yaml

```diff
@@ -1141,7 +1142,6 @@ actions:
   types:
   - container
   releases:
-  - bullseye
```

Comment:
Not tested, but I am afraid that debian/bullseye (variant default) will not get an IPv4 because `systemd-networkd.socket` is not enabled.

Reply:
You're right.

Reply:
Fix: https://github.com/lxc/lxc-ci/pull/505